spec/models/attorney_case_review_spec.rb

```diff
@@ -466,7 +466,28 @@
           expect(attorney_task.appeal.attorney_case_reviews).to eq [case_review]
           expect(attorney_task.appeal.judge_case_reviews).to eq []
         end
-        it "should check for erroneous AMA states"
+
+        it "updates judge task with attorney notes" do
+          label = COPY::ATTORNEY_TASK_NOTES_PREFIX
+          expect(judge_task.reload.instructions).not_to include a_string_matching(/#{Regexp.escape(label)}/)
+          expect(judge_task.reload.instructions).not_to include a_string_matching(/#{note}/)
+          subject
```

Comment:
```suggestion

          subject
                    
```

There is [some precedent](https://github.com/department-of-veterans-affairs/caseflow/blob/master/spec/models/tasks/translation_task_spec.rb#L10-L20) for surrounding `subject` with newlines, though we're not entirely consistent about it and apparently don't have anything set up to enforce it.

My reasoning here is that calling `subject` is sort of the main event of this play. Giving it space highlights that it's doing something major in terms of the test, and makes more of a distinction of the assertions that happen before vs. after the call.